compiler/rustc_target/src/spec/riscv64_linux_android.rs

```diff
@@ -9,7 +9,7 @@ pub fn target() -> Target {
         options: TargetOptions {
             code_model: Some(CodeModel::Medium),
             cpu: "generic-rv64".into(),
-            features: "+m,+a,+f,+d,+c,+Zba,+Zbb,+Zbs".into(),
+            features: "+m,+a,+f,+d,+c,+Zba,+Zbb,+Zbs,+V".into(),
```

Comment:
I think these need to be lower case.

```suggestion
            features: "+m,+a,+f,+d,+c,+zba,+zbb,+zbs,+v".into(),
```

https://github.com/rust-lang/rust/blob/130ff8cb6c3d62ed66daf652cbb5323d3f93c4fc/compiler/rustc_codegen_ssa/src/target_features.rs#L255-L257

The current one generates "recognized feature" warnings.

```
'+Zba' is not a recognized feature for this target (ignoring feature)
'+Zbb' is not a recognized feature for this target (ignoring feature)
'+Zbs' is not a recognized feature for this target (ignoring feature)
'+V' is not a recognized feature for this target (ignoring feature)
```

https://godbolt.org/z/cEq1d6vK6

Reply:
...Wow that's annoying.

Reply:
Done

Reply:
Thank you both.